layouts/partials/clients.html

```diff
@@ -1,20 +1,21 @@
-
 {{ $data := index site.Data site.Language.Lang }}
 
 {{ if $data.clients.clients.enable }}
-    {{ with $data.clients.clients }}
-        <div class="relative max-w-7xl mx-auto px-4 sm:px-6">
-            <h2 class="font-bold text-2xl sm:text-4xl text-center mt-12 sm:mt-24">
-                <span>
-                    {{ with .title }} {{ index (split . " ") 0 | safeHTML }} {{ end }}
-                    <span class="text-[#1D65A6]">{{ with .title }} {{ after (len (index (split . " ") 0)) . | safeHTML }} {{ end }}</span>
-                </span>
-            </h2>
-            <ul class="mt-10 sm:grid grid-cols-5 grid-flow-col gap-20 items-center">
-                {{ range .clients_item}}
-                    <li class="mt-10 flex justify-center"><img class="h-10 sm:h-auto" src="{{ .image | absURL }}" alt="{{ .name }}"></li>
-                {{ end }}
-            </ul>
-        </div>
-    {{ end }}
+{{ with $data.clients.clients }}
+<div class="relative max-w-7xl mx-auto px-4 sm:px-6">
+    <h2 class="font-bold text-2xl sm:text-4xl text-center mt-12 sm:mt-24">
+        <span>
+            {{ with .title }} {{ index (split . " ") 0 | safeHTML }} {{ end }}
+            <span class="text-[#1D65A6]">{{ with .title }} {{ after (len (index (split . " ") 0)) . | safeHTML }} {{ end
+                }}</span>
+        </span>
+    </h2>
+    <ul class="mt-10 sm:grid grid-cols-5 grid-flow-col gap-20 items-center">
+        {{ range .clients_item}}
+        <li class="mt-10 flex justify-center"><img class="h-10 sm:h-[auto]" src="{{ .image | absURL }}"
```

Comment:
Could the `<img>` be nested between the `<li>` tags. Then the class, src, and alt could be on 1 line?